package.json

```diff
@@ -46,7 +46,7 @@
     "browzers": "1.2.0",
     "bulk-require": "0.2.1",
     "mocha": "~1.16.2",
-    "phantomjs": "1.9.19",
+    "phantomjs-prebuilt": "^2.1.4",
```

Comment:
Whoops, that's what I get for simply doing `npm install/remove --save-dev` without looking closely at the changes. Fixed in https://github.com/defunctzombie/zuul/commit/eb2c28547104bad3fd614758f7ff4f6d09ffeb79